packages/api2teams/src/resources/project-template/README.md

```diff
@@ -17,19 +38,99 @@ In the src folder, there are 4 sub folders:
 * commands folder
     - This folder contains bot command handlers, when user send messages, it will match the api, and send back request adaptive card. There is a special command handler which named `helpCommandHandler` which is used to show help messages.
 
+* index.ts file
+    - Index file using `restify` as web service framework to host bot server, and register command handlers and action handlers for bot to handle use messages and button clicking event. 
+
 ![](./images/project-structure.png)
 
```

Comment:
Can we exclude the images from user disk

Reply:
sure